declarative-examples/simple-examples/credentialsMixedEnvironment.groovy

```diff
@@ -2,10 +2,13 @@ pipeline {
   environment {
     // environment variables and credential retrieval can be interspersed
     SOME_VAR = "SOME VALUE"
+    // this assumes that "cred1" has been created on Jenkins Credentials
     CRED1 = credentials("cred1")
     INBETWEEN = "Something in between"
+    // this assumes that "cred2" has been created in Jenkins Credentials
     CRED2 = credentials("cred2")
-    OTHER_VAR = "OTHER VALUE"
+    // Env variables can refer to other variables as well
+    OTHER_VAR = "${SOME_VAR}"
```

Comment:
Here is the output of running this. You can see that the value for `OTHER_VAR` is the same as `SOME_VAR`

https://gist.github.com/HRMPW/7752165ca9a14999c140a29d52405a3a